src/cljam/algo/pileup.clj

```diff
@@ -35,34 +35,51 @@
   [^long start ^long end alns]
   (seq-step start end 0 alns))
 
-(defn- quals-at-ref
-  [idx ^String qual]
-  (let [empty-qual? (and (= (.length qual) 1)
-                         (= (.charAt qual 0) \*))]
-    (if empty-qual?
-      (vec (repeat (count idx) 93)) ;; \~
-      (mapv (fn [[_ x]]
-              (if (number? x)
-                (qual/fastq-char->phred-byte (.charAt qual x))
-                93))
-            idx))))
+(defn- two-vec-transducer
```

Comment:
Just a suggestion, I guess `unzip` and `transpose` are commonly used for this kind of operation. How about `unzip-2`?

Reply:
Renamed to `unzip-2`. Thanks!